hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestDataGenerator.java

```diff
@@ -161,6 +161,17 @@ public static void writePartitionMetadata(FileSystem fs, String[] partitionPaths
     }
   }
 
+  public static List<HoodieRecord> newHoodieRecords(int n, String time) throws Exception {
```

Comment:
can we not use the existing methods in the data generator to write these tests?

Reply:
I copied this method from the `TestCopyOnWriteActionExecutor` class.  Let me check if I can generalize this.

Reply:
I see TestRawTripPayload(String jsonData) is being used at many places. Guess we can just move this method to HoodieTestDataGenerator.

Reply:
Looks like the method is different from other methods in the generator.  I'll keep it as is.

Reply:
but i actually find it surprising we needed to add one. This class as-is is a medley of different individual methods. I would really like to avoid overloading this more.

Reply:
After some investigation, I found that tests in `TestCopyOnWriteActionExecutor` use a different record schema, causing the bulk insert to fail using records from `HoodieTestDataGenerator::generateInserts`.  For the bulk insert tests, I use the same schema provided by `HoodieTestDataGenerator` and `generateInserts()` for generating the testing records.  Now the tests work.